gradle/libs.versions.toml

```diff
@@ -10,6 +10,8 @@ dagger = "2.46.1"
 lifecycle = "2.6.1"
 dokka = "1.8.20"
 mavenPublish = "0.25.3"
+nibelConventionPlugin = "1.0.0"
```

Comment:
let's replace the version here with something like `"ignored", since the version is not applied for convention plugins.